policyengine_us/variables/gov/states/ms/tax/income/taxable_income/ms_taxable_income_joint.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class ms_taxable_income_joint(Variable):
+    value_type = float
+    entity = Person
+    label = "Mississippi taxable income when married couple file jointly"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=13",
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80105228.pdf",  # Line 38 - 49,
+    )
+    defined_for = StateCode.MS
+
+    def formula(person, period, parameters):
+        # MS allowes negative taxable income when married couple file jointly
+        ms_taxable_income_joint_head_or_spouse = person(
+            "ms_taxable_income_joint_head_or_spouse", period
+        )
+        any_spouse_negative_income = person.tax_unit.any(
+            ms_taxable_income_joint_head_or_spouse < 0
+        )
+
+        # 1. both head and spouse have positive taxable income (includes 0)
+
+        # 2. at least one head or spouse has negative taxable income
+        # assign total net_income to tax unit head
+        is_head = person("is_tax_unit_head", period)
+        total_taxable_income = sum(ms_taxable_income_joint_head_or_spouse)
```

Comment:
should this be summed to one person? 

I would use:
total_taxable_income_attributed_to_head = person.tax_unit.sum(ms_taxable_income_joint_head_or_spouse) * is_head

Reply:
yes it should summed to one person. 

`total_taxable_income_attributed_to_head = person.tax_unit.sum(ms_taxable_income_joint_head_or_spouse) * is_head` still will cause the error.  It supposed to be an array by this leads to a matrix

<img width="1000" alt="Screen Shot 2024-04-14 at 8 20 56 PM" src="https://github.com/PolicyEngine/policyengine-us/assets/87620347/cd42ac89-436c-488f-9709-0d654d41c554">